### PR TITLE
ENG-4006 add placeholder epc text

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

File: microfrontends/app-builder-menu/src/components/MenuUI.tsx [hunk 1]
```diff
@@ -76,6 +76,17 @@ const MenuCmp = styled.menu`
...
+const StyledPlaceholder = styled.a`
+  color: ${COLORS.text};
+  font-size: 14px;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 400;
+  line-height: 26px;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+`;
```
Comment (Dv92) [inline microfrontends/app-builder-menu/src/components/MenuUI.tsx]: improve the style with the underline on the link only in the hover state

File: microfrontends/app-builder-menu/src/components/MenuUI.tsx [hunk 2]
```diff
@@ -450,9 +461,17 @@ export function MenuUI(props: Props): JSX.Element {
...
-            </ListGroupItem>
-          )}
+              })
+            ) : (
+              <StyledPlaceholder
+                href="https://hub.entando.com/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Install your first Entando Platform Capability
```
Comment (Dv92) [inline microfrontends/app-builder-menu/src/components/MenuUI.tsx]: we should translate also this content, here are the translations:

**it**: Installa la tua prima Entando Platform Capability
**pt**: Instale sua primeira Entando Platform Capability

please use always content.ts file when you insert new text.

Comment (ichalagashvili) [reply microfrontends/app-builder-menu/src/components/MenuUI.tsx]: done ✅

File: microfrontends/app-builder-menu/src/components/MenuUI.tsx (hunk 1 above)
Comment (ichalagashvili) [reply microfrontends/app-builder-menu/src/components/MenuUI.tsx]: done ✅

**Review by Dv92 — APPROVED** [review]

**Pull request merged**